charts/elasticsearch-umbrella/charts/elasticsearch-statefulset/templates/configmap.yaml

```diff
@@ -3,10 +3,22 @@ apiVersion: v1
 data:
   {{- toYaml .Values.global.elastic_config | nindent 2 }}
   {{- toYaml .Values.elastic_config | nindent 2 }}
+  {{- if semverCompare ">=7.0.0-0" .Values.image.tag }}
+  {{- if .Values.isMaster }}
+  cluster.initial_master_nodes: {{ $replicas := int (toString (.Values.replicaCount)) }}
```

Comment:
In Elastic 7.x this is an extra configuration that is needed, it contains the name of all the master nodes, it needs it for the autodiscovery.